match2/commons/starcraft_starcraft2/match_group_input_starcraft_ffa.lua

```diff
@@ -95,6 +109,77 @@ function FFA.adjustData(match)
 	return match
 end
 
+function FFA.get_pbg(match)
+	local pbg = {}
+
+	local advancecount = tonumber(match.advancecount or 0) or 0
```

Comment:
```suggestion
	local advancecount = tonumber(match.advancecount or 0)
```

Pretty sure this will give the same result

Reply:
it will give nil if `match.advancecount` is a string that is not a number
example:
![Screenshot 2021-07-08 16 59 41](https://user-images.githubusercontent.com/75081997/124944902-fd12da80-e00d-11eb-8f81-886b85ce761d.png)

Reply:
I see, cool

Reply:
I just use `tonumber(match.advancecount) or 0`